sscep.c

```diff
@@ -619,7 +619,7 @@ main(int argc, char **argv) {
 			printf("%s: CA certificate written as %s\n",
 				pname, c_char);
 			(void)fclose(fp);
-			pkistatus = SCEP_PKISTATUS_SUCCESS;
```

Comment:
I'm currently unable to test this patch, it looks fine overall, but I am a bit unsure about this line.
Doesn't this result in a missing status return code?